### ISSUE TITLE
friction-less installation of alerts/collectors

### ISSUE THREAD

**Issue by odyslam** [netdata/community#36]
We should provide a tool (e.g a bash script) so that users can frictionlessly install community collectors and alerts. 

## Collectors

- python.d
- go.d
- charts.d

go.d is tricky, as the user will have to rebuild the go.d binary and replace the existing one. Moreover, we should *somehow* make netdata **NOT** to replace the edited go.d with every nightly release. 

## Alerts

Just copy the alert file in the proper directory. Since we will have multiple different alerts for the same collectors, we should have a structure where people can install specific alerts for a component. 

e.g the `goroutines` alert for `geth`. 

This could be done via a repo structure as: `alerts/geth/goroutines.conf`

## UX

It should be as effortless as possible. For the users to use the community repo, we expect them to be more technical, since they are pushing the boundaries of what netdata can do.

**Comment by odyslam** [comment]
Open question regarding go.d collectors, as any community collector means that the source files of the go.d plugin are edited and the go.d binary is built with that community plugin included. 

That means that netdata has a different go.d than it should, which will lead to it be replaced next time netdata is updated, replacing it with the common go.d plugin. 

Current architecture means that in order to have go.d plugin, the user must disable automatic updates, or with every update, rebuild and replace the go.d plugin with it's own customized that has the community collector. 

More information by @ilyam8

**Comment by andrewm4894** [comment]
I wonder if we could use something like cookiecutter to let a user install a community collector from the repo and it could kinda just walk them through it. 

https://cookiecutter.readthedocs.io

**Comment by ilyam8** [comment]
The simplest form:

 - download file(s) from the community repo
 - move them to appropriate directories
 - restart Netdata

A bash script will do for this case.

**Comment by odyslam** [comment]
yeah, I agree that bash script is a great MVP to gauge user interest. 

I would also suggest to do it in tandem with a Marketplace type of collectors/alerts: https://github.com/odyslam/netdata-marketplace
